dbe/actions/actions.py

```diff
@@ -76,8 +80,13 @@ def slot_mappings(self) -> Dict[Text, Union[Dict, List[Dict]]]:
     @classmethod
     def required_slots(cls, tracker: Tracker) -> List[Text]:
         slots = super().required_slots(tracker)
-        if tracker.get_slot("province_display") and tracker.get_slot("school"):
-            slots = ["confirm_details"] + slots
+        if tracker.get_slot("returning_user") == "yes":
+            if tracker.get_slot("change_details"):
+                slots = ["province", "school", "school_confirm", "confirm_details"]
+            elif tracker.get_slot("confirm_details") == "no":
+                return ["change_details"]
+            elif not tracker.get_slot("confirm_details"):
+                return ["confirm_details"]
```

Comment:
This is all super confusing, but I couldn't figure out a better way of doing this using Rasa Forms